management/virtualmachinedisk/entities.go

```diff
@@ -6,3 +6,12 @@ import "github.com/MSOpenTech/azure-sdk-for-go/management"
 type DiskClient struct {
 	client management.Client
 }
+
+type HostCachingType string // see "HostCachingType*"" constants
```

Comment:
Comments not really necessary as it's obvious :-) and thats not really a godoc comment either